chart/Chart.yaml

```diff
@@ -11,6 +11,7 @@ description: |
 name: rancher-externalip-webhook
 version: %VERSION%
 appVersion: %APP_VERSION%
+kubeVersion: < 1.21.2
```

Comment:
```suggestion
kubeVersion: < 1.21.2-0
```

Reply:
Actually meant `1.22.0` but was using 1.21 for testing :woman_facepalming:

Reply:
Oh, I was assuming 1.21.2 was when it was introduced upstream. I suppose that date doesn't matter.